### DISCUSSION TITLE
笔记状态类型修改

### DISCUSSION THREAD

**Discussion by ZhangWeilong** [Tokinx/Wing#72] [想法]
笔记在发布的时候选择私密或者是正常后，无法修改状态类型。
建议可以修改私密和正常类型。

**Comment by Tokinx** [comment]
> 笔记在发布的时候选择私密或者是正常后，无法修改状态类型。 建议可以修改私密和正常类型。

thanks, 我已经把这个idea创建了issue，等有时间我会开发这个功能。

**Comment by Tokinx** [comment]
0.7.27更新已支持修改笔记类型，同时支持已归档笔记恢复为Publish、Private

**Reply by ZhangWeilong** [reply]
good
